<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="lnEdtUrlTb1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter URL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lnEdtSvLocationTb1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter save location</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDownload">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBrowseTb1">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>100</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="photo.qrc">
        <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarTb1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>441</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>671</width>
        <height>351</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>One Video</string>
       </attribute>
       <widget class="QLineEdit" name="lnEdtVdoUrlTb2">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>40</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lnEdtVdoSvLocationTb2">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>80</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Enter save location</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOneVdoDownload">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>230</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnBrowseVdoTb2">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>80</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCheckVdo">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>130</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Video Quality:  </string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxTb1">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>130</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBarVdoTb1">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>491</width>
          <height>16</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="lblSingleVdoTimeRemaining">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>240</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Full Playlist</string>
       </attribute>
       <widget class="QComboBox" name="comboBoxPlaylist">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>130</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>240</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>360</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1080</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="lnEdtSvPlaylistTb2">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>80</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Enter save location</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lnEdtPlaylistTb2">
        <property name="geometry">
         <rect>
          <x>81</x>
          <y>40</y>
          <width>491</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter  Playlist URL</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnPlaylistDownload">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>250</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnPlaylistSaveBrowse">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>80</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/browse.png</normaloff>:/icons/browse.png</iconset>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBarPlaylist">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>210</y>
          <width>491</width>
          <height>16</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Current Video:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>170</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Full Playlist:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="currentLcdNumber">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="playlistLcdNumber">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>160</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>130</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Video Quality:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblPlaylistTimeRemaining">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>260</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
